src/main/java/com/google/api/codegen/viewmodel/RequestObjectParamView.java

```diff
@@ -30,6 +34,14 @@
 
   public abstract boolean isArray();
 
+  @Nullable // Used in C#
+  public abstract String defaultValue();
+
+  // Work around a limitation in the snippet language
+  public String defaultValue(String template) {
+    return defaultValue() != null ? String.format(template, defaultValue()) : "";
+  }
```

Comment:
I'd prefer a more direct approach. Add a method "hasDefaultValue" and use an `@if` in the snippet to switch the output based on the value.